build-logic/convention/src/main/kotlin/cz/matee/devstack/plugin/AndroidApplicationConventionPlugin.kt

```diff
@@ -45,7 +45,7 @@ class AndroidApplicationConventionPlugin : Plugin<Project> {
                 }
 
                 buildFeatures {
-                    buildConfig = true
+                    buildConfig = false
```

Comment:
V AGP 8.0 je false default, takže klidně můžeme smazat

Reply:
Ja by som to tam nechala, aby keď niekto bude hľadať, kde to zapnúť, nájde to ľahšie, môže byť?

Reply:
Ok, klidně 😄